src/js/view/controls/components/tooltip.js

```diff
@@ -5,30 +5,16 @@ define([
 ], function(Events, utils, _) {
 
     return class Tooltip {
-        constructor(name, ariaText, ariaShown, elementShown) {
+        constructor(name, ariaText, elementShown) {
             _.extend(this, Events);
             this.el = document.createElement('div');
             let className = 'jw-icon jw-icon-tooltip ' + name + ' jw-button-color jw-reset';
             if (!elementShown) {
                 className += ' jw-hidden';
             }
+            utils.addAriaLabel(this.el, ariaText);
+
             this.el.className = className;
-            if (ariaText) {
-                this.el.setAttribute('tabindex', '0');
-                this.el.setAttribute('role', 'button');
-                this.el.setAttribute('aria-label', ariaText);
-            }
-            if (ariaShown === true) {
-                // Avoiding to hide the tooltip if requested
-                // e.g. The volume tooltip overlay don't work
-                // with the keyboard but can still mute/unmute
-                this.el.setAttribute('aria-hidden', 'false');
-            } else {
-                // Tooltip only works on :hover :^(
-                // It is not working using the keyboard
-                // Hiding it for ARIA while not supported
-                this.el.setAttribute('aria-hidden', 'true');
```

Comment:
So this was stopping the aria-label and tabbing to occur for HD, CC & Audio Tracks button. So this gets makes them accessible when you click on them, even if you can't access the functionality while tabbing,